src/test/java/com/example/rets_api/service/PropertyServiceTest.java

```diff
@@ -90,4 +90,27 @@ public void shouldGetPropertyByParams_getPropertiesByParams(){
         checkAllFields_Property(propertyEntity, propertyDTOListResponse.get(0));
     }
 
+    @Test
+    public void shouldDeletePropertyById_deletePropertyById(){
+        //Creating objects
+        PropertyEntity propertyEntity = createPropertyEntityResponseAndID(Long.valueOf(123456789));
+
+        //Mocking calls
+        when(propertyRepositoryJPA.findById(any())).thenReturn(Optional.of(propertyEntity));
```

Comment:
you need to add another mock for the call propertyRepositoryJPA.delete.

Reply:
done

Reply:
the mock when is a method void is done like this doNothing().when(propertyRepositoryJPA).delete(any());

Reply:
thanks, modified it